File: app/src/main/java/com/test/test/models/Parameters.java
package com.test.test.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Parameters {

   @SerializedName("AWIPSidentifier")
   List<String> AWIPSidentifier;

   @SerializedName("WMOidentifier")
   List<String> WMOidentifier;

   @SerializedName("NWSheadline")
   List<String> NWSheadline;

   @SerializedName("BLOCKCHANNEL")
   List<String> BLOCKCHANNEL;


    public void setAWIPSidentifier(List<String> AWIPSidentifier) {
        this.AWIPSidentifier = AWIPSidentifier;
    }
    public List<String> getAWIPSidentifier() {
        return AWIPSidentifier;
    }
    
    public void setWMOidentifier(List<String> WMOidentifier) {
        this.WMOidentifier = WMOidentifier;
    }
    public List<String> getWMOidentifier() {
        return WMOidentifier;
    }
    
    public void setNWSheadline(List<String> NWSheadline) {
        this.NWSheadline = NWSheadline;
    }
    public List<String> getNWSheadline() {
        return NWSheadline;
    }
    
    public void setBLOCKCHANNEL(List<String> BLOCKCHANNEL) {
        this.BLOCKCHANNEL = BLOCKCHANNEL;
    }
    public List<String> getBLOCKCHANNEL() {
        return BLOCKCHANNEL;
    }
    
}

File: app/src/main/java/com/weather/alerts/models/Parameters.kt
package com.weather.alerts.models

import com.google.gson.annotations.SerializedName

class Parameters {
    @SerializedName("AWIPSidentifier")
    var aWIPSidentifier: List<String>? = null

    @SerializedName("WMOidentifier")
    var wMOidentifier: List<String>? = null

    @SerializedName("NWSheadline")
    var nWSheadline: List<String>? = null

    @SerializedName("BLOCKCHANNEL")
    var bLOCKCHANNEL: List<String>? = null
}